appcelerator/titanium_mobile
# fix(ios): top section can disappear in grouped ListView

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28109

**Summary:**
When a `ListView` with "style" set to `GROUPED` or `INSET_GROUPED` is configured and the top-most section has not title or views, then that top-most section will disappear when tapping on a `SearchBar` or relayout such as an orientation change.

**Test:**
1. Build and run the below on iOS.
2. Tap on the search bar.
3. Verify that space above the first section does not disappear.
4. Rotate to landscape and verify the space above the section is still there.
5. In the code below, use the `INSET_GROUP` style and rebuild.
6. Repeat steps 2-4.

```javascript
const window = Ti.UI.createWindow();
const listView = Ti.UI.createListView({
	style: OS_IOS ? Ti.UI.iOS.ListViewStyle.GROUPED : undefined,
//	style: OS_IOS ? Ti.UI.iOS.ListViewStyle.INSET_GROUPED : undefined,
	searchView: Ti.UI.createSearchBar(),
	keepSectionsInSearch: true,
	sections: [
		Ti.UI.createListSection({
			items: [
				{ properties: { title: 'Apples', searchableText: 'Apples' } },
				{ properties: { title: 'Bananas', searchableText: 'Bananas' } },
				{ properties: { title: 'Oranges', searchableText: 'Oranges' } },
			],
		}),
		Ti.UI.createListSection({
			items: [
				{ properties: { title: 'Carrots', searchableText: 'Carrots' } },
				{ properties: { title: 'Onions', searchableText: 'Onions' } },
				{ properties: { title: 'Tomatoes', searchableText: 'Tomatoes' } },
			],
		}),
	],
});
window.add(listView);
window.open();
```
